src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1052,6 +1053,21 @@ function ($a) {
                                 ->end()
                             ->end()
                         ->end()
+                        ->scalarNode('default_bus')->defaultValue('default')->end()
```

Comment:
Could we default this to null, then in the extension:

1) If there is 1 extension, just use it
2) If there are multiple, throw a clear exception that the `default_bus` key needs to be defined?

I could see a situation where someone sees:

> The default bus named "default" is not defined. Define it or change the default bus name.

And thinking, where did default come from?

Reply:
Yep, I like that idea 👍

Reply:
Changed it this way.